src/main/resources/db.changelog/changes/init.yaml

```diff
@@ -2,9 +2,28 @@ databaseChangeLog:
   - logicalFilePath: no-path
   - changeSet:
       id: init change set
-      author: ikantsevich@exadel.com
+      author: Alisher
```

Comment:
Alisher I think you can leave this init file and create new one for yourself

Reply:
I have deleted including init file changes in the master changeset. It won't create a test table.